### ISSUE TITLE
[feature] Support ESLint v9

### ISSUE THREAD

**Issue by Lonli-Lokli** [effector/eslint-plugin#164]

**Comment by rtatarinov** [comment]
Do u have an approximately deadline to solve it?

**Comment by StringKe** [comment]
Interim Program Support ESLint v9

```js
import { fixupPluginRules } from "@eslint/compat"
import js from '@eslint/js'
import effector from 'eslint-plugin-effector'
import reactHooks from 'eslint-plugin-react-hooks'
import reactRefresh from 'eslint-plugin-react-refresh'
import globals from 'globals'
import tseslint from 'typescript-eslint'

export default tseslint.config(
  { ignores: ['dist'] },
  {
    extends: [js.configs.recommended, ...tseslint.configs.recommended],
    files: ['**/*.{ts,tsx}'],
    languageOptions: {
      ecmaVersion: 2020,
      globals: globals.browser,
    },
    plugins: {
      'react-hooks': reactHooks,
      'react-refresh': reactRefresh,
      effector: fixupPluginRules(effector),
    },
    rules: {
      ...reactHooks.configs.recommended.rules,
      ...effector.configs.recommended.rules,
      ...effector.configs.react.rules,
      ...effector.configs.future.rules,
      ...effector.configs.patronum.rules,
      'react-refresh/only-export-components': [
        'warn',
        { allowConstantExport: true },
      ],
    },
  },
)
```

**Comment by Grawl** [comment]
```
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"7 || 8" from eslint-plugin-effector@0.15.0
npm ERR! node_modules/eslint-plugin-effector
npm ERR!   dev eslint-plugin-effector@"*" from the root project
```
